bin/check-uptime.rb

```diff
@@ -34,11 +34,22 @@ class CheckUptime < Sensu::Plugin::Check::CLI
          description: 'Warn if uptime is below SEC',
          proc: proc(&:to_i),
          default: 180
+  option :crit,
+         short: '-c SEC ',
+         description: 'Critical if uptime is below SEC',
+         proc: proc(&:to_i),
+         default: 180
```

Comment:
by inserting this default it makes it a breaking change, also I would argue that the default does not make sense to me. I'd opt for no defaults and only check if its > 0 (because `nil.to_i == 0`)